code/game/objects/items/devices/laserpointer.dm

```diff
@@ -134,7 +134,7 @@
 
 	//catpeople
 	for(var/mob/living/carbon/human/H in view(1,targloc))
-		if(!isfelinid(H) || H.incapacitated() || H.is_blind())
+		if(!isfelinid(H) || !ismoth(H) || H.incapacitated() || H.is_blind())

```

Comment:
this should be !(isfelinid(H) || ismoth(H)); as it is now, this code makes neither felinids (since they're not moths) nor moths (since they're not felinids) jump at laser pointers